docs/source/recruitment.rst

```diff
@@ -0,0 +1,234 @@
+sign-on page. It takes a comma-separated list of qualification names to
+avoid. In order to prevent participant from repeating an experiment or group,
+you can set this parameter to an experiment ID or group name, and set
+``assign_qualifications`` to ``true``.
+
+Waiting Rooms
+^^^^^^^^^^^^^
+
+One other thing that affects recruitment is the use of a :doc:`waiting room
+<waiting_rooms>`. Waiting rooms are used when an experiment requires
+participants to be synchronized. Participants are kept in the "room" until
+enough of them have signed up and are ready to start. Experimenters can set
+the ``quorum`` in the experiment code.
+
+Recruitment Handling in Experiment Code
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In addition to the previously mentioned configuration parameters, Dallinger
+experiment creators can use their experiment code to further affect
+recruitment. There are a number of basic recruitment attributes that can be
+set on experiment initialization, and recruitment can be further affected by
+calling specific methods during experiment runtime.
+
+There are specific points in an experiment code where recruitment is usually
+affected. To show how you can set up recruitment for your experiment, we
+will use GridUniverse code as a guide. The methods discussed here are part
+of the experiment base class, so it is not required to implement them in
+your experiment, but most experiments need at least the ``configure`` and
+``create_network`` methods.
+
+::
+
+    def configure(self):
+        super(Griduniverse, self).configure()
+        self.num_participants = config.get('max_participants', 3)
+        self.quorum = self.num_participants
+        self.initial_recruitment_size = config.get('num_recruits',
+                                                   self.num_participants)
+
+The ``configure`` method is called during experiment initialization, and is
+where experiment specific configuration takes place. Many times,
+configuration parameters from the experiment `config.txt` file are used
+here.
+
+GridUniverse defines ``max_participants`` and ``num_recruits`` parameters.
+They are used in this method to set ``experiment.num_participants``,
+``experiment.quorum`` and ``experiment.initial_recruitment_size``. The first
+of these is only used in GridUniverse code, so we can ignore it.
+
+In its ``configure`` method, GridUniverse sets ``experiment_quorum`` to be
+the same as the configured number of participants. This means that the
+participants will be held in the waiting room until all participants have
+been recruited. Other experiment designs might not need all of the
+participants to be ready at the same time, but only a fraction of them. This
+attribute only applies to experiments that use a waiting room. The default
+value for ``experiment.quorum`` is zero (no waiting room).
+
+``experiment.initial_recruitment_size`` is the number of participants
+required at experiment start. This is used at experiment launch to start the
```

Comment:
"at experiment start" might not be wrong, however "at the start of the experiment" sounds better to my ears. 
This could be purely subjective though :)
Also in the next sentence you mention "experiment launch". Are you referring the same thing?